lib/utils/functionArgumentsSearch.js

```diff
@@ -21,7 +25,10 @@ module.exports = function(source /*: string*/, functionName /*: string*/, callba
 			target: functionName,
 			functionNames: 'check',
 		},
-		(match) => {
+		(/** @type {{
+			startIndex: number,
+			endIndex: number,
+		}} */ match) => {
```

Comment:
@hudochenkov I have provided types to `style-search` instead of providing type to callback